l2tscaffolder/scaffolders/timesketch.py

```diff
@@ -59,6 +59,20 @@ def _GeneratePluginTest(self) -> str:
     return self._mapping_helper.RenderTemplate(
         self.TEMPLATE_PLUGIN_TEST, self.GetJinjaContext())
 
+  def AddEntriesToInitFiles(self) -> Iterator[Tuple[str, str]]:
+    """Returns a list of init files that were modified.
+
+    Adds an entry into the import section of an __init__.py file
+    in the correct alphabetical order.
+
+    Yields:
+      tuple (str, str): file name of source and destination.
+    """
+    plugin_string = 'from {0:s} import {1:s}\n'.format(
```

Comment:
To make this clearer, please separate the separator replacement from the string substitution.